src/allmydata/introducer/client.py

```diff
@@ -35,11 +38,10 @@ def __init__(self, tub, introducer_furl,
         self._sequencer = sequencer
         self._cache_filepath = cache_filepath
 
-        self._my_subscriber_info = { "version": 0,
-                                     "nickname": self._nickname,
-                                     "app-versions": [],
```

Comment:
I'm guessing you had a merge conflict to resolve here.  I think `app-versions` should remain in this dict with an empty list as its value.  This was a consequence of https://github.com/tahoe-lafs/tahoe-lafs/pull/907.

Reply:
Yeah, must've done it wrong.